package.json

```diff
@@ -9,6 +9,8 @@
     "build": "mkdir -p dist && babel src -d dist",
     "test": "LOGGER_SILENT=1 mocha --compilers js:babel-register -r should -r src/utils/setup-jsdom -R ${REPORTER-dot} 'src/*.spec.js'",
```

Comment:
Could you please add `npm run js:lint` here to make it check lint errors on `npm run test`?